### PR TITLE
Harmonize use of operators

### PR DESCRIPTION
**Pull request by sosna**
Dear all,

Another PR, this time focusing on the agreement around operators.

The data page has not changed in this PR, because the necessary changes have been made in [another PR](https://github.com/sdmx-twg/sdmx-rest/pull/124).

Cheers,

Xavier

### CODE REVIEW

File: v2_1/ws/rest/docs/4_1_introduction.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # SDMX RESTful API
 
-## A Brief Introduction to REST
+This SDMX API is based on the REST principles
 
-This SDMX API is based on the REST principles, as described below:
+- In REST, specific information is known as **Resource**. In SDMX, specific resources would be, for example, code lists, concept schemes, data structure definitions, dataflows, etc. Each resource is addressable via a **global identifier** (i.e.: a URI).
```
Comment (dosse) [inline v2_1/ws/rest/docs/4_1_introduction.md:5]: Suggestion: ... In SDMX, specific resources would be **_maintainable artefacts_**, for example, ...

File: v2_1/ws/rest/docs/4_3_structural_queries.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Furthermore, a keyword may be used:
 
 Keyword | Scope | Description
 --- | --- | ---
-all | itemID | Returns all items belonging to the item scheme
+`*` | itemID | Returns all items belonging to the item scheme
 
 The following rules apply:
```
Comment (dosse) [inline v2_1/ws/rest/docs/4_3_structural_queries.md:86]: Line 159: should be  http://ws-entry-point/structure/codelist/BIS,ECB/CL_FREQ

**Review by dosse — COMMENTED** [review]
Comments on other files:  

4_4_data_queries.md:  
- line 18: "*,FLOW_ID,+" should be "*,FLOW_ID,**~**", and "AGENCY_ID,FLOW_ID,+" should be "AGENCY_ID,FLOW_ID,**~**"
- line 18: the comma separation doesn't make sense anymore (since comma means "or", but if I'm not wrong then this change will be addressed in another SDMX 3.0 feature.)
- lines 30-35: change "all" to "*"

4_6_1_other_queries.md:
- line 15: "*,FLOW_ID,+" should be "*,FLOW_ID,**~**", and "AGENCY_ID,FLOW_ID,+" should be "AGENCY_ID,FLOW_ID,**~**"
- line 16: change "+" to ","
- lines 28-36: change "all" to "*"

File: v2_1/ws/rest/docs/4_1_introduction.md (hunk 1 above)
Comment (sosna) [reply v2_1/ws/rest/docs/4_1_introduction.md]: Done, thanks @dosse

File: v2_1/ws/rest/docs/4_3_structural_queries.md (hunk 2 above)
Comment (sosna) [reply v2_1/ws/rest/docs/4_3_structural_queries.md:86]: Done. Thanks for spotting this, @dosse :)

**Comment by sosna** [comment]
Thanks, @dosse. I have taken care of 2 out of your 4 comments.

Regarding the other 2:

- Changes in 4_4_data_queries.md are taken care of in [cr-117-1](https://github.com/sdmx-twg/sdmx-rest/pull/124);
- Changes in 4_6_1_other_queries.md are taken care of in [cr-117-3](https://github.com/sdmx-twg/sdmx-rest/pull/126).

**Review by dosse — APPROVED** [review]
ok, great, thanks a lot!

**Comment by sosna** [comment]
Thanks, @dosse!

**Pull request merged**